### PR TITLE
Document CMake build steps on Windows

### PR DESCRIPTION
**Pull request by geographika**
Update docs to include CMake build steps on Windows using Visual Studio, relating to #1040
Temporary output HTML to show formatting at: http://www.geographika.net/swig/Windows.html#Windows_cmake
Comments, suggestions, changes welcome.

### CODE REVIEW

File: Doc/Manual/Windows.html [hunk 1]
```diff
@@ -364,6 +365,89 @@ <H4><a name="Windows_building_alternatives">3.3.1.3 Building swig.exe alternativ
...
+
+<H4><a name="Windows_building_alternatives">3.3.1.3 Building swig.exe alternatives</a></H4>
+
+
+<p>
+If you don't want to install Cygwin or MinGW, use a different compiler to build 
+SWIG. For example, all the source code files can be added to a Visual C++ project 
+file in order to build swig.exe from the Visual C++ IDE.
+</p>
+
```
Comment (wsfulton) [inline Doc/Manual/Windows.html:387]: Duplicate of above, should be deleted.

**Review by wsfulton — COMMENTED** [review]
If nuget is suggested to install pcre, shouldn't the recommendation be to use it to install bison too?

I'd expect chocolatey to be the easiest way to install the two dependencies. If you are able to test that and it works, it would be great to have these two simple ways documented for installing the dependencies, namely, chocolatey and nuget.

File: Doc/Manual/Windows.html (hunk 1 above)
Comment (geographika) [reply Doc/Manual/Windows.html]: Sorry about that - removed

**Comment by geographika** [comment]
> If nuget is suggested to install pcre, shouldn't the recommendation be to use it to install bison too?

Yes that would make sense - I've updated (and tested) so nuget is used for bison, CMake, and pcre. 
 
> I'd expect chocolatey to be the easiest way to install the two dependencies. If you are able to test that and it works, it would be great to have these two simple ways documented for installing the dependencies, namely, chocolatey and nuget.

I couldn't find PCRE on Chocolatey - https://chocolatey.org/packages?q=pcre I tried searches for Perl etc. and nothing relevant came up.

**Comment by wsfulton** [comment]
Looks good, thanks. I'll merge when #1040 is complete.

**Pull request merged**